src/Bridge/Doctrine/Orm/Filter/ExistsFilter.php

```diff
@@ -190,4 +226,15 @@ private function isAssociationNullable(array $associationMapping): bool
 
         return true;
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function extractProperties(Request $request/*, string $resourceClass*/): array
```

Comment:
I think you don't need to add this since this filter has not been used in this version yet.